helm-chart/flink-operator/README.md

```diff
@@ -28,6 +29,11 @@ The instructions to install the Flink operator chart:
     helm install --name [RELEASE_NAME] . --set operatorImage.name=[IMAGE_NAME]
     ```
 
+    If you're installing the chart into a preexisting namespace, make sure to set the namespace name and instruct the chart not create one like so:  
+
+    ```bash
+    helm install --name [RELEASE_NAME] . --set operatorImage.name=[IMAGE_NAME],flinkOperatorNamespace.name=[NAMESPACE_NAME],flinkOperatorNamespace.name=false
```

Comment:
Seems it should be `flinkOperatorNamespace.create=false` instead of `....name=...`.

Reply:
Right, I've fixed that.